dev/infra_server_objects_count_collector.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+usage()
+{
+cat << EOF
+usage: $0 options
+
+This script gets the count of the chef server objects.
+
+OPTIONS:
+   -h      Shows the help message
+   -S      Chef Server url (Eg: https://chef-server.example.com)
+   -K      Path to the pivotal user key
+   -F      Output file path
+EOF
+}
+
+check_binary_install_jq() {
+  if ! which "$1" > /dev/null; then
+    # Using a subshell to redirect output to stderr. It's cleaner this way and will play nice with other redirects.
+    echo "jq is required to execute the script. Intalling it on this machine now.."
```

Comment:
We should prompt the user about the unavailability of jq.
Installing `jq` will be tricky as different platforms have their standard way of deployment